lib/manager/components/NotesViewer.js

```diff
@@ -69,14 +69,14 @@ export default class NotesViewer extends Component<Props> {
                 <Media key={note.id}>
                   <Left>
                     <img
-                      width={64}
+                      alt={note.userEmail}
                       height={64}
                       src={`${this._getUserUrl(note.userEmail)}`}
-                      alt={note.userEmail} />
+                      width={64} />
                   </Left>
                   <Body>
                     <Panel className='comment-panel'>
-                      <Panel.Heading>
+                      <Panel.Heading><Panel.Title componentClass='h3'>
                         <Heading>
```

Comment:
See same remarks regarding headings and link.

Reply:
Repaired in 566bef2e3b25f7411ef8ad1d0183d5763c185b39